lib/sidekiq_unique_jobs/locksmith.rb

```diff
@@ -312,7 +312,7 @@ def brpoplpush(conn, wait)
     # @api private
     #
     def rpoplpush(conn)
-      conn.rpoplpush(key.queued, key.primed)
+      conn.lmove(key.queued, key.primed, "RIGHT", "LEFT")
```

Comment:
should we rename the method name too?

Reply:
yes, thanks!!

Reply:
thanks for the suggestion!! It got me thinking that if we change the method name, we would also need to change the params it receives in order to be compliant with what `lmove` does. Do you think it is worthy to do these changes?

Reply:
@GabrielaGuedes I'm not the best to say. Hopefully a maintainer can chime in. 
I'm just grateful someone is solving this annoying warning